config-model/src/main/java/com/yahoo/vespa/model/admin/clustercontroller/ClusterControllerContainer.java

```diff
@@ -108,6 +110,11 @@ public void getConfig(BundlesConfig.Builder builder) {
         }
     }
 
+    @Override
+    public void getConfig(PlatformBundlesConfig.Builder builder) {
+        bundles.forEach(builder::bundles);
```

Comment:
Will the final config produced contain the changes from same method in `ContainerCluster`?

Reply:
Yes, this is a container, which is a child of the cluster. The new method is just a copy of the method above that produced the old config.